tests/test_kci.py

```diff
@@ -1,19 +1,22 @@
 import numpy as np
 from flaky import flaky
-from sklearn.metrics.pairwise import linear_kernel, polynomial_kernel, rbf_kernel
+from sklearn.metrics.pairwise import rbf_kernel
 
 from pywhy_stats import kci
 from pywhy_stats.kernels import delta_kernel
 
 ####################################################
 # Unconditional independence tests - Continuous
 ####################################################
+seed = 12345
+np.random.seed(seed)
 
 
 @flaky(max_runs=3)
```

Comment:
If we work with a fixed see, no need for flaky decorator. Note that the fixed seed are generating consistently the same values across different OS and python versions etc.

Reply:
Is the suggestion then to remove random seed and keep flaky I suppose?

Reply:
I am actually also fine with a fixed seed (as long as it doesn't impact downstream tests). However, I still see the benefit of having different generated data every run to avoid optimizing for particular set of data. For instance, 200 samples for the KCI test is already quite small and might fail with a different seed. However, is this now due to a subtle issue in the implementation or just a general statistical issue?

Reply:
I'll just remove the seed for now and we can revisit the rng vs seed vs flaky option in the unit-tests if it is an issue